### ISSUE TITLE
Add 15 projects in September

### ISSUE THREAD

**Issue by AdrienLemaire** [awesome-global-contributions/awesome-global-contributions#29]
Due: 2019/09/30 #7

Given I only added 1 project to the list in August, I'll try to build the new habit and add at least 15 projects in September.

- [x] Safecast
- [x] OpenFarm
- [x] Alaveteli
- [x] Kolibri
- [x] Union VMS
- [x] MediaWiki
- [ ] OnSSET
- [ ] [OSeMOSYS](http://www.osemosys.org/get-started.html) related to OnSSET

**Comment by OscarAblinger** [comment]
Awesome, thank you :)

**Comment by AdrienLemaire** [comment]
Spending too much time investigating projects that don't fit our requirements or can't be added at the moment and waiting for action. @AblingerOscar do you have a good idea on how to manage this information?

- [Xanthos](https://github.com/JGCRI/xanthos)
    - Anthos is an open-source hydrologic model, written in Python, designed to quantify and analyze global water availability. 
    - mail sent to ask for a roadmap and contribution guide. the community could, for example, build and maintain a public browsing app for anyone to go and visualize global water availability over time.
- [Audacious](https://audaciousproject.org/)
    - big ideas projects that can have a huge global impact
    - mail sent to request more details regarding licensing. Too difficult to find the OSS projects in the lot atm.
- [PrivacyNet](https://privacy.net/)
   - The analyzer tool teaches visitors about web security issues and best practices.
   - Tweet sent to request open sourcing of the project
- [Atlas](http://enablingthefuture.org/)
   - Tweet sent to request a contribution guide. SDG10
- [WikiGender](https://www.wikigender.org/)
   - Tweet sent to request fixing the login process on their site
- OpenForis
   - Tweet sent to request a contribution guide. SDG 15
- OHorizons
   - Tweet sent to ask how can contributions help them. SDG6
- [OnSSET](http://www.onsset.org/)
   - mail asked to ask for online tools to be fixed and to update the contributions guidelines

**Comment by OscarAblinger** [comment]
I initially thought about just posting them in a slack channel, but thinking more about it, we could just simply make a new branch for them?
We could add a new branch where we have only projects that are still missing stuff we require.
I'd probably make them a markdown file with an explanation (and links) as to why it's not eligible and potentially a issue/tweet/etc. that we opened to rectify it. The yaml object could then be added to the markdown as a code tag, if some of it was already created.
PRs for that branch would probably clutter it too much, so I'd not require them from maintainers and hold them to a lower standard than our master branch.

Once they complete what was necessary, we'd create the yaml file based on the master branch, create a PR, audit it there (no need to do it earlier than that) and delete the mardown file on the other branch with a link to the PR in the commit message.

I'm not sure it's the _best_ solution, but it would avoid us having too many open issues and loosing the overview (like it would likely happen in a slack channel) and have it on Github ready for everyone to see and potentially even add to it.

What do you think?

If we decide to go with that idea, I'd probably base that branch on commit cb47af8 [(browse files)](https://github.com/awesome-global-contributions/awesome-global-contributions/tree/cb47af8b7926ee32c19f71918597b2dfcbe5ef53)

**Comment by AdrienLemaire** [comment]
having multiple main git branches for this project sounds a bit too much.
For the moment, I feel that showing that we're doing anything, whether it makes it to the list or not, would be the most efficient and boost the team morale. I wrote a message about it on Slack.

Ended up investigating 14 projects, with 6 ending up in the awesome list. This was done in 2 weeks, and I burnt out the rest of the month. Didn't get a single reply to my emails and tweet, even after bumping them up.
